docker-compose.yml

```diff
@@ -70,6 +71,8 @@ services:
     depends_on:
       - redis
       - postgres
+    volumes:
+      - /mnt/geoserver_resources:/geoserver_resources
```

Comment:
this is the primary change -- we allow the celery worker to RW the geoserver resources